packages/amazonq/src/lsp/lspInstaller.ts

```diff
@@ -22,21 +27,27 @@ export class AmazonQLspInstaller extends BaseLspInstaller.BaseLspInstaller<
     protected override async postInstall(assetDirectory: string): Promise<void> {
         const resourcePaths = this.resourcePaths(assetDirectory)
         await fs.chmod(resourcePaths.node, 0o755)
+        if (await fs.exists(resourcePaths.ripGrep)) {
+            await fs.chmod(resourcePaths.ripGrep, 0o755)
+        }
     }
 
     protected override resourcePaths(assetDirectory?: string): AmazonQResourcePaths {
         if (!assetDirectory) {
             return {
                 lsp: this.config.path ?? '',
                 node: getNodeExecutableName(),
+                ripGrep: `ripgrep/${getRgExecutableName()}`,
```

Comment:
need to make sure this is a signed binary so that OS security mechanism does not alert on this

Reply:
I tried using it already from the packaged servers.zip and there was no alert